.gitignore

```diff
@@ -1,2 +1,3 @@
 bin/configlet
 bin/configlet.exe
+.precomp
```

Comment:
Perl 6 creates a directory with this name automatically alongside a module when it is used. In our case, it will be created alongside each module whenever someone runs the tests.

I don't know much about Perl 6 precompilation yet unfortunately, but this directory will show up every time one of the modules gets used. The same line can be found in the .gitignore files for various Perl 6 modules for this reason.

I can add a comment if it helps.